src/style.css

```diff
@@ -1,3 +1,66 @@
 body {
   margin: 0;
+  padding: 0;
+  
+  --activation-color: #00ACDC;
 }
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  width: 100vw;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0;
+  padding: 0;
```

Comment:
instead of using for inner elements
```suggestion
  padding: 0 50px;
```

Reply:
still not resolved
![image](https://github.com/mate-academy/layout_moyo-header/assets/55200530/777e2b75-5034-4fbe-8b58-8319642e4bb9)